pyart/graph/radardisplay.py

```diff
@@ -1019,6 +1050,44 @@ def _label_axes_vpt(self, axis_labels, ax):
         else:
             ax.set_ylabel(y_label)
 
+    def _set_vpt_time_axis(self, ax, dForm=None, tz=None,
+                           date_MinTicker=None):
+        """ Set the x axis as a time formatted axis.
+
+        Parameters
+        ----------
+        ax : Matplotlib axis instance
+            Axis to plot. None will use the current axis.
+        dForm : str
+            Format of the time string for x-axis labels.
+        tz : str
+            Time zone info to use when creating axis labels (see datetime).
+        date_MinTicker : str
+            Sting to set minor ticks of date axis,
+            'second','minute','hour','day' supported.
+        """
+        if dForm is None:
+            dForm = '%H:%M'
+        if date_MinTicker is None:
+            date_MinTicker = 'minute'
+
+        # Set the date format
+        date_Fmt = DateFormatter(dForm, tz=tz)
+
+        # Set x-axis and tick parameters
+        ax.xaxis.set_major_formatter(date_Fmt)
+        if date_MinTicker == 'second':
+            ax.xaxis.set_minor_locator(SecondLocator())
+        elif date_MinTicker == 'minute':
+            ax.xaxis.set_minor_locator(MinuteLocator())
+        elif date_MinTicker == 'hour':
+            ax.xaxis.set_minor_locator(HourLocator())
+        elif date_MinTicker == 'day':
+            ax.xaxis.set_minor_locator(DayLocator())
+
```

Comment:
I'm not seeing any minor ticks in my plot, might have something incorrect but it would be good to tests this.  I'm also not apposed to not adding minor ticks and leaving this customization to the users.